cloud-info/requirements.txt

```diff
@@ -1,5 +1,6 @@
-# Cloud info version is 9d4c4c516b9311c77564444cb9ecbb059b7f2192
-git+https://github.com/EGI-Federation/cloud-info-provider.git@9d4c4c516b9311c77564444cb9ecbb059b7f2192
+# Cloud info version is 43cefc204b3e07211c6c37df2ee20eab845c3428
+# 43cefc204b3e07211c6c37df2ee20eab845c3428 includes json glue support
+git+https://github.com/EGI-Federation/cloud-info-provider.git@43cefc204b3e07211c6c37df2ee20eab845c3428
```

Comment:
reference tags (release) instead of hashes?

Reply:
Ideally yes, but we don't release that often in cloud-info-provider, no specific reason for that though

Reply:
No worries - I suggest semantic-release later. Happy to provide PR to that repo later!